src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -403,6 +403,7 @@ void CoilCoolingDXCurveFitPerformance::size(EnergyPlus::EnergyPlusData &state)
         }
         this->mySizeFlag = false;
     }
+    this->oneTimeAvailSchedSetup(state);
 }
```

Comment:
Moved this new function `oneTimeAvailSchedSetup()` to `size()` from `simulate()` because it needs to be executed only once.